README.md

```diff
@@ -1,2 +1,260 @@
+
+➕ It has fractal (Hausdorff) dimension of:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;D=2"/>
+
+🎨 The easiest way to create a visualization of the Mandelbrot set is to plot it on a computer, using per-pixel operations. To do so, you can convert a complex number, <img src="https://latex.codecogs.com/svg.latex?\large&space;c=a+bi"/>, to pixel coordinates on an image, where a is the X position of the pixel and b is the Y position of the pixel.\
+A complex c belongs to the Mandelbrot set if the function:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;f_{c}(z)=z^2+c"/>
+
+does not diverge with z starting at 0 when under infinite recursion.\
+By applying this recursion to every complex number that corresponds to each pixel on an image, and coloring it based on the number of iterations it takes to diverge, you can get an image similar to the one shown on the gif.\
+All pixel coordinates should be normalized to not exceed |2| on any axis, since the entirety of the Mandelbrot set is contained inside a circle of radius 2.
+
+### Julia Set
+
+![screenshot](./media/julia_set_1.png)
+![screenshot](./media/julia_set_2.png)
+![screenshot](./media/julia_set_3.png)
+
+📌 Let:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;f_{c}(z)=z^2+c"/>
+
+where z and c are numbers on the complex plane. The Julia set can then be defined as the set of complex numbers z that under infinite iteration by f<sub>c</sub> do not diverge.\
+c can be any complex number, as long as it does not change during recursion. For each different c there is a different set o z's that follow the described rule.\
+The set is named after the French mathematician Gaston Julia
+
+➕ It has fractal (Hausdorff) dimension of:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;D=2"/>
+
+🎨 The Julia set can't be plotted as one single image, due to the infinite possible values of c. Because of that, you must first choose a value of c you want to plot for.\
+The rest of the process is essentialy the same as the one used to plot the Mandelbrot set: by transforming complex numbers to pixel coordinates, you can iterate f<sub>c</sub> for each complex number corresponding to each pixel on the image and color it based on the number of iterations it takes to diverge.\
+Pixel coordinates to plot the Julia fractals should also be normalized to no exceed |2| on any axis, just like with the Mandelbrot set.\
+By changing the complex c chosen, you can get different images. Interestingly, if c ∈ Mandelbrot set, the corresponding Julia fractal will be one solid shape (refer to screenshot 2), and if c ∉ Mandelbrot set, the corresponding Julia fractal will not be just one solid shape, but a collection of disjointed shapes (refer to screenshot 1).
+
+## 3D Fractals
+
+### Sierpinski Tetraheadron
```

Comment:
Aparentemente, o certo é "Tetrahedron" em inglês, não sei se foi erro de digitação ou alguma curiosidade histórica sobre o tema, mas enfim, se for mudar, tem q lembrar de trocar em tudo aqui, no sumário e na referência do sumário tbm, pra n quebrar td.